src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/HSDB.java

```diff
@@ -1104,9 +1104,6 @@ public void addAnnotation(Address addr, OopHandle handle) {
                           } else if (region.isHumongous()) {
                             anno = "Humongous ";
                             bad = false;
-                          } else if (region.isPinned()) {
-                            anno = "Pinned ";
-                            bad = false;
```

Comment:
Does this mean that the region will now always be one of Free, Young, Humongous, or Old?